great_expectations/expectations/expectation_configuration.py

```diff
@@ -130,9 +130,9 @@ def __init__(  # noqa: PLR0913
         expectation_context: Optional[ExpectationContext] = None,
         rendered_content: Optional[List[RenderedAtomicContent]] = None,
     ) -> None:
-        if not isinstance(expectation_type, str):
+        if not isinstance(type, str):
```

Comment:
It makes me very sad that this has `type: str` and we're still doing an `isinstance`, but removing that should be in a separate PR I suppose